runtime/src/feature_set.rs

```diff
@@ -38,6 +42,7 @@ lazy_static! {
         (pico_inflation::id(), "pico-inflation"),
         (spl_token_v2_multisig_fix::id(), "spl-token multisig fix"),
         (bpf_loader2_program::id(), "bpf_loader2 program"),
+        (compute_budget_config2::id(), "compute budget configuration 2"),
```

Comment:
Why "2"?

Reply:
there was an original compute budget config and there will be more as time goes on

Reply:
How about "1ms compute budget configuration", that's what `ComputeBudget::default()` now is I think

Reply:
There might be tweaks etc.. that might also align with 1ms, not sure we want to try and name each tweak we make, numeric versioning seems to be more straight forward

Reply:
I feel like we could continue to come up with more meaningful names for future budget tweaks. 

I'm thinking that the output from `solana feature status` is also for general 3rd party developer/validator consumption.
Somebody asking about the status of "1ms compute budget configuration" is going to be much more useful than "compute budget configuration 2" (especially if there's a 2, 3, 4, 5 as well).